.github/workflows/ci.yml

```diff
@@ -53,6 +53,22 @@ jobs:
         # https://github.com/golangci/golangci-lint-action/issues/244
         skip-cache: true
 
+  tidy:
+    name: Tidy Emulator
```

Comment:
Why is there a tidy job only for the emulator?
Btw `make emulator-build` is only there to make sure the emulator can build. It's a very small job that could be added to another job.

Reply:
I grouped both the Tidy and Emulator build steps into a single job because they're both so small. The Tidy job is for the entire code base, not just for the emulator.

Reply:
I created a new job for building the emulator bd55f35f955c9c3eef3dd0cb79db8bd2d9c39ea7

Reply:
I updated the emulator job with a new name and put it back into the same job as the main `Tidy` job
https://github.com/onflow/flow-go/pull/3420/commits/107911a774c50c871dfe21d0239c07a2f2627d19